terraform/modules/website-e2e-test/iam.tf

```diff
@@ -0,0 +1,16 @@
+# IAM permissions
+
+locals {
+  # Cloud build service account roles
+  pubsub_iam_roles_list = [
+    "roles/cloudscheduler.admin"
+  ]
+}
+
+resource "google_project_iam_member" "web_e2e_iam_members" {
+  project  = var.project_id
+  provider = google
```

Comment:
ah just realized this is the reason why my testing failed. Ideally we don't want the terraformer service account to grant itself any permissions.  Instead, let's add an entry for this role in the bootstrap step.  All Terraform roles should be added only in the boostrap.